docs/functions/functions.json

```diff
@@ -538,6 +627,44 @@
 				true
 			]
 		},
+		{
+			"name": "intersects",
+			"id": "intersects",
+			"unary": false,
+			"category": "lists",
+			"role": "function",
+			"description": "Checks if two lists share one or more of the same item.\nFor example, `intersects([1,2,3], [4,3,5])` will be true because both share the item `3`.",
```

Comment:
Avoid using brackets to show examples of lists, they collide with the default expression syntax so wouldn't work if copy/pasted in most cases.